docs/reference/modules/asset.md

```diff
@@ -50,6 +55,12 @@ A source map is not produced at all in a production or production-like environme
 
 This option currently applies only to the admin UI build, not the `ui/src` build.
 
+### `rebundleModules`
```

Comment:
Rebundle modules is 100% supported by Vite.

Reply:
Added a sentence about Vite.